apps/mobile/src/components/TradeChecklistFlow/components/CalculateAmountFlow/atoms.ts

```diff
@@ -0,0 +1,174 @@
+import {atom, type PrimitiveAtom} from 'jotai'
+import {type BtcOrSat, type TradePriceType} from '../../domain'
+import {focusAtom} from 'jotai-optics'
+import * as TE from 'fp-ts/TaskEither'
+import * as T from 'fp-ts/Task'
+import {
+  mainTradeCheckListStateAtom,
+  offerForTradeChecklistAtom,
+} from '../../atoms'
+import {publicApiAtom} from '../../../../api'
+import {pipe} from 'fp-ts/function'
+import {AcceptedCurrency} from '@vexl-next/rest-api/dist/services/btcPrice'
+import reportError from '../../../../utils/reportError'
+import {checkInputValueIsDecimal} from '../../utils'
+
+export const SATOSHIS_IN_BTC = 100000000
```

Comment:
You can always use `_` in number definition to make it more readable. `... = 100_000_000`